src/Microsoft.VisualStudio.ProjectSystem.Managed/Tree/Dependencies/DependencyTreeFlags.cs

```diff
@@ -88,27 +88,23 @@ public static class DependencyTreeFlags
                 + GenericDependency;
 
         internal static readonly ProjectTreeFlags TargetNode = ProjectTreeFlags.Create("TargetNode");
-        internal static readonly ProjectTreeFlags SubTreeRootNode = ProjectTreeFlags.Create("SubTreeRootNode");
+        internal static readonly ProjectTreeFlags SubTreeRootNode = ProjectTreeFlags.Create(ProjectTreeFlags.Common.VirtualFolder) +
```

Comment:
This should be named as `SubTreeRootNodeFlags` for consistency with other fields.

Reply:
It should be the reverse; Flags should be removed from the names. We know they are flags based on the type name. Feel free to follow up with that change - I've didn't change the name.

Reply:
Members with `Flags` suffix are composite. Those without are singular.